packages/cactus-plugin-odap-hermes/src/test/typescript/integration/backup-gateway-after-client-crash.test.ts

```diff
@@ -0,0 +1,907 @@
+    const contractDir = path.join(__dirname, contractRelPath);
+
+    // ├── package.json
+    // ├── src
+    // │   ├── assetTransfer.ts
+    // │   ├── asset.ts
+    // │   └── index.ts
+    // ├── tsconfig.json
+    const sourceFiles: FileBase64[] = [];
+    {
+      const filename = "./tsconfig.json";
+      const relativePath = "./";
+      const filePath = path.join(contractDir, relativePath, filename);
+      const buffer = await fs.readFile(filePath);
+      sourceFiles.push({
+        body: buffer.toString("base64"),
+        filepath: relativePath,
+        filename,
+      });
+    }
+    {
+      const filename = "./package.json";
+      const relativePath = "./";
+      const filePath = path.join(contractDir, relativePath, filename);
+      const buffer = await fs.readFile(filePath);
+      sourceFiles.push({
+        body: buffer.toString("base64"),
+        filepath: relativePath,
+        filename,
+      });
+    }
+    {
+      const filename = "./index.ts";
+      const relativePath = "./src/";
+      const filePath = path.join(contractDir, relativePath, filename);
+      const buffer = await fs.readFile(filePath);
+      sourceFiles.push({
+        body: buffer.toString("base64"),
+        filepath: relativePath,
+        filename,
+      });
+    }
+    {
+      const filename = "./asset.ts";
+      const relativePath = "./src/";
+      const filePath = path.join(contractDir, relativePath, filename);
+      const buffer = await fs.readFile(filePath);
+      sourceFiles.push({
+        body: buffer.toString("base64"),
+        filepath: relativePath,
+        filename,
+      });
+    }
+    {
+      const filename = "./assetTransfer.ts";
+      const relativePath = "./src/";
+      const filePath = path.join(contractDir, relativePath, filename);
+      const buffer = await fs.readFile(filePath);
+      sourceFiles.push({
+        body: buffer.toString("base64"),
+        filepath: relativePath,
+        filename,
+      });
+    }
+
+    const response = await apiClient.deployContractV1({
+      channelId,
+      ccVersion: "1.0.0",
+      sourceFiles,
+      ccName: fabricContractName,
+      targetOrganizations: [org1Env, org2Env],
+      caFile: `${orgCfgDir}ordererOrganizations/example.com/orderers/orderer.example.com/msp/tlscacerts/tlsca.example.com-cert.pem`,
+      ccLabel: "basic-asset-transfer-2",
+      ccLang: ChainCodeProgrammingLanguage.Typescript,
+      ccSequence: 1,
+      orderer: "orderer.example.com:7050",
+      ordererTLSHostnameOverride: "orderer.example.com",
+      connTimeout: 60,
+    });
+
+    const { packageIds, lifecycle, success } = response.data;
+    expect(response.status).toBe(200);
+    expect(success).toBe(true);
+    expect(lifecycle).not.toBeUndefined();
+
+    const {
+      approveForMyOrgList,
+      installList,
+      queryInstalledList,
+      commit,
+      packaging,
+      queryCommitted,
+    } = lifecycle;
+
+    Checks.truthy(packageIds, `packageIds truthy OK`);
```

Comment:
@AndreAugusto11 Jest has truthiness assertions [1] built in and I would recommend using those instead of the `Checks` class which is meant for production code to validate its preconditions. They kind of produce the same end result here, technically, but with `Checks` the developer experience will be worse because if these assertions fail it will look like as if the test code itself had crashed due to a bug **in the test code itself** not the code that it is verifying to be working properly (very nuanced but important difference when you are trying to debug a failing test case and all you have to go on is some obscure exception message ;-))

[1] https://jestjs.io/docs/using-matchers#truthiness

Reply:
I also changed this in the other tests where I was using`Checks`